src/openssh/Makefile

```diff
@@ -0,0 +1,26 @@
+.ONESHELL:
+SHELL = /bin/bash
+.SHELLFLAGS += -e
+
+MAIN_TARGET = openssh-server_$(OPENSSH_VERSION)_$(CONFIGURED_ARCH).deb
+DERIVED_TARGETS = openssh-server-dbgsym_$(OPENSSH_VERSION)_$(CONFIGURED_ARCH).deb
+
+$(addprefix $(DEST)/, $(MAIN_TARGET)): $(DEST)/% :
+	# Obtain openssh: https://salsa.debian.org/ssh-team/openssh/-/tree/debian/1%257.9p1-10+deb10u2
+	rm -rf ./openssh-server
+	git clone https://salsa.debian.org/ssh-team/openssh.git openssh-server
+	pushd ./openssh-server
+	git checkout -b openssh-src -f debian/1\%$(OPENSSH_VERSION)
```

Comment:
>debian/1\%$(OPENSSH_VERSION) [](start = 32, length = 28)

The commit ID is better than tag because it is not movable and reproducible.
@xumia Please help check.

Reply:
Talk with Xuhui offline. I will use a commit id here for better reproducible and add a comment to specific the tag name